lab-jonathan/server.js

```diff
@@ -0,0 +1,56 @@
+let http = require('http');
+let fs = require('fs');
+let parseBody = require('./lib/parse-body.js');
+let url = require('url');
+let querystring = require('querystring');
+let cowsay = require('cowsay');
+let PORT = process.env.PORT || 3000;
+
+
+let server = http.createServer(function(req, res){
+  let path = url.parse(req.url);
+  let query = querystring.parse(path.query);
+  if(req.method === 'GET' && path === '/'){
+    res.writeHead(200, {'Content-Type': 'text/plain'});
+    res.write(cowsay.say({text: 'hello world'}));
+    res.end();
+  }
+
+  if(req.method === 'GET' && path.pathname === '/cowsay'){
+    if(path.query){
```

Comment:
Given that you're checking to see if there's any `query.message` to use, probably best to validate `if(query){}` rather than path.query.